.pre-commit-config.yaml

```diff
@@ -88,3 +88,12 @@ repos:
     hooks:
       - id: djlint-django
         files: '^rocky/.*/templates/.*$'
+
+  - repo: https://github.com/thibaudcolas/pre-commit-stylelint
```

Comment:
By default, the hook will run stylelint only for  `.css`, `.sass`, `.scss`. Is additional path filtering really necessary?

Reply:
It's not that much of a big deal, but it serves as a bit of documentation and there's really no need to scan the entire tree with almost 8000 source files if you can filter and point it in the right direction

Reply:
Updated. More like this?

Reply:
Now that I've checked the hook implementation I realized, as you have already mentioned, it indeed only runs on stylesheets. I was under the assumption that it also runs on HTML templates (as advertised by the docs), hence the suggestion for the templates folder filter.

In that case it might be better to use the `rocky/assets/src/` (and `docs/`) folder instead of `rocky/*/templates/*`

Reply:
Updated.